File: java/program18.java
import java.util.*;

class Number
{
    public void Display(int iNo)
    {
        if(iNo < 0)
        {
            iNo = -iNo;
        }
        
        for(int iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print(iCnt+"\t");
        }
        System.out.println();
    }
}

class program18
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Frequency");
        int iValue = sobj.nextInt();

        Number obj = new Number();
        obj.Display(iValue);
    }
}

File: cpp/program18.cpp
// program20.c

#include<iostream> 
using namespace std;

class Number
{
    public:
      int iNo;

    Number(int X)
    {
        iNo = X;
    }

    void Display()      
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            cout<<iCnt<<"\n";
        }
    }
        
};

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout<<"Enter first number :"<<"\n";
    cin>>iValue;

    Number nobj(iValue);
    nobj.Display();

    return 0;
}